.github/workflows/conda_and_docs_build.yml

```diff
@@ -0,0 +1,85 @@
+name: conda_and_docs_build
+
+on:
+  release:
+    types: [published]
+  push:
+    branches: [ master ]
+    tags:
+      - '**'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  conda_build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: conda-build
+      uses: paskino/conda-package-publish-action@v1.4.2
```

Comment:
shall we bump to [version 1.4.3](https://github.com/paskino/conda-package-publish-action/tree/v1.4.3)